src/SecretManager/Microsoft.DncEng.SecretManager/SecurityAuditLogger.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Runtime.CompilerServices;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Audit.Geneva;
+
+namespace Microsoft.DncEng.SecretManager
+{
+    /// <summary>
+    /// SecurityAuditLogger is a class that is used to log security audit events to the security event log and Geneva
+    /// </summary>
+    public class SecurityAuditLogger
+    {
+        private ILogger ControlPanelLogger;
```

Comment:
```suggestion
        private ILogger ControlPlaneLogger;
```
I think that's what you meant?